File: app/src/main/java/com/example/bangabandhuplay/data/model/category/root/single/ContentSource.java
package com.example.bangabandhuplay.data.model.category.root.single;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class ContentSource {

    @SerializedName("key")
    @Expose
    private Object key;
    @SerializedName("value")
    @Expose
    private Object value;

    public ContentSource(Object key, Object value) {
        this.key = key;
        this.value = value;
    }

    public Object getKey() {
        return key;
    }

    public Object getValue() {
        return value;
    }
}

File: app/src/main/java/com/durbar/bangabandhuplay/data/model/search_content/ContentSource.kt
package com.durbar.bangabandhuplay.data.model.search_content

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class ContentSource(
    @field:SerializedName("key")
    val key: Any? = null,
    @field:SerializedName("value")
    val value: Any? = null
)
